docs/tables/scaleway_registry_image.md

```diff
@@ -0,0 +1,36 @@
+# Table: scaleway_registry_namespace
+
+Namespaces allow you to manage your Container Registry in a simple, clear and human-readable way.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  id,
+  status,
+  created_at,
+  tags,
+  size
+from
+  scaleway_registry_image;
+```
+
+### List images updated in last 10 days for a repository
+
+```sql
+select
+  name,
+  id,
+  status,
+  created_at,
+  updated_at
+  tags,
+  size
+from
+  scaleway_registry_image
+where
+  updated_at >= now() - interval '10' day
+```
```

Comment:
@jplanckeel the examples look good 👍 but they seem a bit light. Could you please add a few more examples like `List images which are public` or `Group images based on the status`? Such examples are simple and are usually good starting points for users who are looking to use the tables.

Reply:
Hi, i have add an other exemple with visibility. I have look other provider (aws, docker) and i don't find example to inspire me